### PR TITLE
Bump sdk

### PR DESCRIPTION
**Pull request by sisuresh**
### What

1. Bumps the sdk
2. Removes testutils (I don't understand yet why this used to work, but if you run make test, you'll see a failure about a missing testutils feature.)

### CODE REVIEW

**Review by anupsdf — APPROVED** [review]
LGTM

**Comment by anupsdf** [comment]
The `make test` step in `build-and-test` job completed fine. Not sure why it stopped at the post action [rust-cache step](https://github.com/stellar/soroban-examples/actions/runs/4792939912/jobs/8524933697?pr=235).

**Comment by sisuresh** [comment]
> The `make test` step in `build-and-test` job completed fine. Not sure why it stopped at the post action [rust-cache step](https://github.com/stellar/soroban-examples/actions/runs/4792939912/jobs/8524933697?pr=235).

I ran the action again to see if the last run was just a fluke.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+*/*/target/
+*/target/
 /target/
```
[the remark this answers is not in the document]
Comment (sisuresh) [reply .gitignore]: I rebased on top of 236

File: account/Makefile [hunk 2]
```diff
@@ -4,7 +4,6 @@ all: test
 
 test: build
 	cargo test
-	cargo test --features testutils
```
Comment (leighmcculloch) [inline account/Makefile:7]: 👍🏻 Thanks.

**Review by leighmcculloch — APPROVED** [review]

**Pull request merged**